src/modules/vendor/router/routes.js

```diff
@@ -275,6 +275,18 @@ const routes = [
         path: 'trainers/info',
         name: 'trainers info',
         component: () => import('src/modules/vendor/pages/trainers/administrative/Info'),
+        beforeEnter: async (to, from, next) => {
+          try {
+            if (['trainers courses', 'trainers courses info'].includes(from.name)) {
+              to.query.defaultTab = 'contracts';
```

Comment:
Pourquoi as tu defini "Contrat" comme etant l'ordre de mission par defaut ?

Reply:
c'était pour que comme en front quand on clique sur voir mes formations et qu'on arrive sur le kanban, quand on revient en arrière on est de nouveau sur contrat. Sauf qu'il y a un menu en moins côté formateur donc ça le fait aussi quand on passe direct du kanban au infos. Je le retire

Reply:
ESt-ce qu'il y aurait pas un autre moyen d'arriver sur l'onglet contrat quand on navigue du tableau ordre vers le kanban puis qu'on revient ? C'est dommage d'arriver sur l'onglet info

Reply:
je pense pas, les from et to sont identiques dans les 2 cas, je ne vois pas de possibilité de les différencier